ui/components/app/collectibles-detection-notice/collectibles-detection-notice.js

```diff
@@ -9,54 +9,104 @@ import {
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import Button from '../../ui/button';
 import { EXPERIMENTAL_ROUTE } from '../../../helpers/constants/routes';
 import { setCollectiblesDetectionNoticeDismissed } from '../../../store/actions';
+import { ENVIRONMENT_TYPE_FULLSCREEN } from '../../../../shared/constants/app';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
 
 export default function CollectiblesDetectionNotice() {
   const t = useI18nContext();
   const history = useHistory();
+  const environmentType = getEnvironmentType();
 
   return (
-    <Box marginBottom={4} className="collectibles-detection-notice">
+    <Box className="collectibles-detection-notice">
       <Dialog type="message" className="collectibles-detection-notice__message">
-        <button
-          onClick={() => setCollectiblesDetectionNoticeDismissed()}
-          className="collectibles-detection-notice__message__close-button"
-          data-testid="collectibles-detection-notice-close"
-        />
-        <Box display={DISPLAY.FLEX}>
-          <Box paddingTop={2}>
-            <i style={{ fontSize: '1rem' }} className="fa fa-info-circle" />
+        {environmentType !== ENVIRONMENT_TYPE_FULLSCREEN && (
+          <button
+            onClick={() => setCollectiblesDetectionNoticeDismissed()}
+            className="fas fa-times collectibles-detection-notice__message__close-button"
+            data-testid="collectibles-detection-notice-close"
+          />
+        )}
+        <Box display={DISPLAY.FLEX} width={BLOCK_SIZES.MAX}>
+          <Box paddingTop={1}>
+            <i
+              style={{ fontSize: '1rem', color: '#037DD6' }}
+              className="fa fa-info-circle"
+            />
           </Box>
-          <Box paddingLeft={4}>
+          <Box paddingLeft={2}>
             <Typography
               color={COLORS.BLACK}
               align={TEXT_ALIGN.LEFT}
-              variant={TYPOGRAPHY.H6}
+              variant={TYPOGRAPHY.H7}
               fontWeight={FONT_WEIGHT.BOLD}
             >
               {t('newNFTsDetected')}
             </Typography>
-            <Typography
-              color={COLORS.BLACK}
-              align={TEXT_ALIGN.LEFT}
-              variant={TYPOGRAPHY.H6}
-              boxProps={{ marginBottom: 4 }}
-            >
-              {t('newNFTsDetectedInfo')}
-            </Typography>
-            <Button
-              type="link"
-              onClick={() => {
-                history.push(EXPERIMENTAL_ROUTE);
-              }}
-              className="collectibles-detection-notice__message__link"
-            >
-              {t('selectNFTPrivacyPreference')}
-            </Button>
+            {environmentType === ENVIRONMENT_TYPE_FULLSCREEN ? (
+              <Box
+                display={DISPLAY.INLINE_FLEX}
+                flexDirection={FLEX_DIRECTION.ROW}
+              >
+                <Typography
+                  color={COLORS.BLACK}
```

Comment:
This PR should be merged today but used the correct design token colors. I'll keep working on the documentation

```suggestion
                  color={COLORS.TEXT_DEFAULT}
```